module-1/src/Components/MovieForm/MovieForm.tsx

```diff
@@ -0,0 +1,120 @@
+import "./MovieForm.css";
+interface IMovieForm {
+  action: string;
```

Comment:
Instead of building the title here you can pass it fully as a string.

```suggestion
  title: string;
```

Reply:
fixed